Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -350,7 +359,9 @@ def retrieve_versions_by_arch(formula_or_cask:, repositories:, args:, name:)
           current_version_value = Version.new(loaded_formula_or_cask.version)
         end
 
-        livecheck_latest = livecheck_result(loaded_formula_or_cask)
+        livecheck_latest, livecheck_latest_throttled = livecheck_result(loaded_formula_or_cask)
+        # TODO: Pass down `livecheck_latest` info to print output for throttled formulae or casks
+        livecheck_latest = livecheck_latest_throttled if livecheck_latest_throttled
 
         new_version_value = if (livecheck_latest.is_a?(Version) && livecheck_latest >= current_version_value) ||
```

Comment:
Doesn't need cleaned up in this PR but: I hate that it's using different types here to indicate success/failure.